NinjaResidence/DirectX.h

```diff
@@ -196,4 +197,7 @@ class DirectX {
 	HRESULT InitD3d(HWND hWnd, LPCSTR FilePath);
 	HRESULT InitDinput(HWND hWnd);
 	HRESULT DeviceState = NULL;
+
+	const int MaxKeyNumber = 256;
```

Comment:
```suggestion
	static const int MaxKeyNumber = 256;
```
ってして`m_KeyOldState`とかより上に書いておけば、
```
	BYTE m_KeyOldState[MaxKeyNumber];
	BYTE m_KeyState[MaxKeyNumber];
```
みたいにマジックナンバー消せますよ。

Reply:
指摘の通りに変更しました